requirements.txt

```diff
@@ -4,7 +4,7 @@ pandas>=0.23.4
 scipy>=1.1.0
 SoundFile>=0.10.2
 torch>=1.3.0
-pytorch-lightning>=0.7.5,<0.8
+pytorch-lightning>=0.8
```

Comment:
wouldn't be better to use == because new lightning releases tend to break things

Reply:
We'll see, release 0.9.0 will be released soon and we can test against it. 
If things break, we'll upgrade to 0.9.0 directly